mews-api.md

```diff
@@ -972,8 +972,8 @@ There are certain rules that need to be followed in order for Mews to process th
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `code` | `string` | required \(always\) | Unique code of the reservation within the whole booking. Any value but `_` accepted. No characters limit. |
-| `spaceTypeCode` | `string` | required \(exc. Cancellation\) | Space type code of the reservation. |
-| `ratePlanCode` | `string` | required \(exc. Cancellation\) | Rate type code of the reservation. |
+| `spaceTypeCode` | `string` | required \(exc. Cancellation\) | Space type code of the reservation. In case ```spaceTypeCode``` missing, a channel manager is expected to add a fallback code to replace it, e.g. ```NOCODE```. Then reservation will be created for a default space category in Mews.|
```

Comment:
```suggestion
| `spaceTypeCode` | `string` | required \(exc. Cancellation\) | Space type code of the reservation. In case ```spaceTypeCode``` missing, a channel manager is expected to add a fallback code to replace it, e.g. ```NOCODE```. Then reservation will be created for a suitable space category in Mews.|
```